### PR TITLE
Home work 05

### PR DESCRIPTION
**Pull request by userhost1**
Привет. На проверку

### CODE REVIEW

File: src/homework05/part2_t18.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+// Найти в строке не только запятые, но и другие знаки препинания. Подсчитать общее их количество.
+
+public class part2_t18 {
+    public static void main(String[] args) {
+        String someText = "Найти в строке не только запятые, но и другие знаки препинания. Подсчитать общее их количество.".trim();
+
+        int count = 0;
+
+        char[] chars = {'.', ',', '!', '?', '-'};
```
Comment (TheFallen058) [inline src/homework05/part2_t18.java:11]: well done but missed some symbols (":",";" etc

File: src/homework05/part2_t21.java [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+        //  1
+        String empty1 = "";
+
+        Date ex1_Start = new Date();
+        for (String s : words) {
+            empty1 += s;
+        }
+
+        Date ex1_Finish = new Date();
```
Comment (TheFallen058) [inline src/homework05/part2_t21.java:163]: Please use System.nanoTime() for such small time periods

**Review by TheFallen058 — APPROVED** [review]
Well done

**Comment by userhost1** [comment]
Thanks for System.nanoTime();

fixed

**Review by TheFallen058 — APPROVED** [review]

**Pull request closed without merging**